prog/kubernetes/kubernetes_cluster_nexus.rb

```diff
@@ -91,7 +118,11 @@ def before_run
   end
 
   label def wait
-    nap 65536
+    k8s_svc_raw = kubernetes_cluster.sshable.cmd("sudo kubectl --kubeconfig /etc/kubernetes/admin.conf get service --all-namespaces --field-selector spec.type=LoadBalancer -ojson")
```

Comment:
nit: we can move this to another function for readability and consistency with other progs.